packages/components/src/components/Dialog/Dialog.tsx

```diff
@@ -206,9 +224,10 @@ DialogContent.displayName = 'Dialog.Content';
 
 export const DialogHeader = (
     {
-        children,
+        padding = 'compact',
```

Comment:
if we set a default value here, wouldn't it will always override the dialog-wide padding?

Reply:
very true, last night i just added the global one and went on my way, nice catch